safe-global/web-core
# fix: shown unknown when current chain not found in config

## What it solves

Resolves missing wallet details in header

## How this PR fixes it

When the wallet is connected to a chance not available in the UI, it now displays "Unknown" instead.

![image](https://user-images.githubusercontent.com/20442784/215110981-0cdb6c95-e023-4516-a315-c2c37df189e4.png)

## How to test it

Connect wallet to UI and switch the wallet chain to one not available in the UI. Observe the wallet displaying with "Unknown" in the header and account centre. Navigate to Safe creation and observe the widget shown "Unknown" as well.

## Screenshots

![image](https://user-images.githubusercontent.com/20442784/215110778-c50e79a1-4359-4a5b-b701-1afdb0c25a29.png)

![image](https://user-images.githubusercontent.com/20442784/215117235-ad801ec9-44fd-42cf-bdef-fdbc03c164fd.png)

![image](https://user-images.githubusercontent.com/20442784/215117264-25ed89d5-258d-48cd-adb2-32bef1233282.png)
